### DISCUSSION TITLE
What versions of PDF does Sumatra support

### DISCUSSION THREAD

**Discussion by hellorunner** [sumatrapdfreader/sumatrapdf#3512] [Q&A]
What versions of PDF (1.5 - 2.0) does Sumatra support?

**Comment by GitHubRulesOK** [comment]
0.0-9.9 and probably others, depending on content, the version is a comment at the start of file regardless of faults
so this one should be a minimum of perhaps 1.5 or 6 or 7 since it has transparency in XObjects
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/2b94be68-a322-4485-94fb-dc03d3877b7e)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/0711108e-6532-4688-84ea-7d039b0e9ca9)
